### ISSUE TITLE
Remove option for - knife vsphere vm vmdk add

### ISSUE THREAD

**Issue by petericebear** [chef-partners/knife-vsphere#449]
### Platform Details
* Version of vSphere: 5.5

### Scenario:
Remove vmdk net to the add method.

Hi,
Is it possible to create an opposite for adding a vmdk to a vm? 
Like: knife vsphere vm vmdk remove XX

**Comment by petericebear** [comment]
I see there is a reconfigeVM option: VirtualMachine.Config.RemoveDisk
But I do not see how to leverage that functionality.

When I do: $ knife vsphere vm disk list name-of-the-virtual-machine
The output is:
  0          Hard disk 1 10.00 GiB
  1          Hard disk 2 50.00 GiB
  2          Hard disk 3 30.00 GiB

Hope that there is a way to remove disk number 2 or 3

**Comment by petericebear** [comment]
Have something working:
https://gist.github.com/petericebear/3640f0b1ee60277e633efe0f36716406

**Comment by swalberg** [comment]
Hi Peter, we don't have such a command. Kinda surprising, really, seems like it would be helpful.

**Comment by petericebear** [comment]
Yeah with that script it checks that you dont delete the primary disk (0), but all the others you can delete with:
knife vsphere vm disk remove vps-name 1

Where 1 is the disk index show with the disk list command.

**Comment by swalberg** [comment]
Oh, I didn't see you had a gist... that looks really good! Do you want to put it up as a PR?

Couple of small requests though: can you use SearchHelper to find the VM? See [here](https://github.com/chef-partners/knife-vsphere/blob/master/lib/chef/knife/vsphere_vm_vmdk_add.rb#L12) and [here](https://github.com/chef-partners/knife-vsphere/blob/master/lib/chef/knife/vsphere_vm_vmdk_add.rb#L48). And instead of `puts` can you use `ui.info`, please?
